### PR TITLE
WIP - adding APCO

### PR DESCRIPTION
**Pull request by tinkeringtuck**
this is a draft/work in progress, but wanted to make it available for visibility

### CODE REVIEW

File: src/opower/utilities/apco.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
+"""Appalachian Power Company (APCO)."""
```
Comment (tronikos) [inline src/opower/utilities/apco.py:1]: Rename here and elsewhere to "Appalachian Power (APCO)"? That's the name I see at https://en.wikipedia.org/wiki/American_Electric_Power and https://www.appalachianpower.com/

File: src/opower/utilities/apco.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            if result and result.group("token"):
+                self.opower_access_token = result.group("token")
+
+    def handle_endtag(self, tag: str) -> None:
+        """Recognizes the end of inline scripts."""
+        if tag == "script":
+            self._in_inline_script = False
+
+
+class APCO(UtilityBase):
```
Comment (tronikos) [inline src/opower/utilities/apco.py:60]: Rename to AppalachianPower. Also rename the file to appalachianpower.py. The convention is to match the utility domain.

File: src/opower/utilities/apco.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+    @staticmethod
+    def subdomain() -> str:
+        """Return the opower.com subdomain for this utility."""
+        return "apco"
+
+    @staticmethod
+    def timezone() -> str:
+        """Return the timezone."""
+        return "America/New_York"
```
Comment (tronikos) [inline src/opower/utilities/apco.py:76]: Update to the correct timezone

**Review by tronikos — CHANGES_REQUESTED** [review]

File: src/opower/utilities/apco.py (hunk 3 above)
Comment (tinkeringtuck) [reply src/opower/utilities/apco.py]: @tronikos I believe this is correct, are you under the impression it should be a different time zone. I live in the New York time zone where I pay Appalachian Power for my electricity, but AEP may own subsidiaries in other time zones.

**Comment by tronikos** [comment]
The library supports all AEP utilities including Appalachian Power. Closing this.

**Pull request closed without merging**